var/spack/repos/builtin/packages/pkgconf/package.py

```diff
@@ -19,6 +19,7 @@ class Pkgconf(AutotoolsPackage):
 
     license("ISC")
 
+    version("2.2.0", sha256="b06ff63a83536aa8c2f6422fa80ad45e4833f590266feb14eaddfe1d4c853c69")
```

Comment:
Flagging for extra eyes. What are changes in 2: series?

Reply:
Ah whoops doubled up on myself. Also submitted this as a stand alone in https://github.com/spack/spack/pull/44008. 
v2 doesn't appear to break API compatibility with anything in CI.